### DISCUSSION TITLE
Add a button combination for  Output Config Mode switching

### DISCUSSION THREAD

**Discussion by Raton-Laveur** [darthcloud/BlueRetro#614] [Ideas]
Regarding [this workaround with Saturn analog controllers](https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-System-Specific-User-Manual#notes), a button combo would make it easier to switch between "Digital" and "Analog" Output Config modes. 

Example: 
- BlueRetro is set on "GamePad" mode, controller is Digital. User can navigate ODE for game selection. 
- User inputs button combo that switches BlueRetro to "GamePadAlt". 
   - if technically necessary, user pushes EN reset button on BlueRetro in order to apply the new config. 
- Controller is now Analog and can be used in-game. 
- When resetting game and going back at ODE game selection, user inputs button combo to switch back to "GamePad" mode. 

Examples of button combinations:
- GamePad mode: 
Main Left Trigger (L2) + Main Right Trigger (R2) + Middle Right (Start) + Face Right (Circle) + D-pad Left
- GamePadAlt mode: 
Main Left Trigger (L2) + Main Right Trigger (R2) + Middle Right (Start)+ Face Right (Circle) + D-pad Right

Switching to keyboard or mouse would be excluded as it's not really possible to input combos - especially with mouse. 

Or is it technically impossible as changes in Output Config require rebooting BlueRetro? Close this discussion if it is the case. Thanks!

**Comment by Raton-Laveur** [comment]
I realize this could also be achieved through [GameID profiling](https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#212---how-to-switch-a-game-to-its-own-config), but I can't find info about how ODEs can send this info to BlueRetro/how BlueRetro reads it so I can send a feature request to Sega Saturn ODE devs.

**Comment by darthcloud** [comment]
The current design of BlueRetro don't allow mode change without a reboot. So this is currently not possible.

**Reply by Raton-Laveur** [reply]
I took in account the reboot limitation with "if technically necessary, user pushes EN reset button on BlueRetro in order to apply the new config" in the example steps though.

**Comment by DJm00n** [comment]
I'll second this thread.
Many PS1 games were released before the DualShock/DualAnalog controller came out. So in those games the sticks just don't work (Crash Bandicoon 1 for example). I can go into BlueRetro settings and switch the mapping for LS->D-Pad (by the way it would be nice to add this as GamePad Alt profile for PSX/PS2), but it's awfully long and inconvenient to do every time I want to play certain PS1 games. It would be great to select the left stick mode with Analog/Mode button (which is roughly what it's for) or any other hotkey.

Even the built-in PS3 emulator has that "Analog Mode" switch with "Digital Mode"/"Analog Mode":
![image](https://user-images.githubusercontent.com/1285934/219951555-1294f8bd-4d96-4e79-b54a-093176564d16.png)
https://youtu.be/IJRBev4ktIM

**Comment by darthcloud** [comment]
Be happy that there a way to do it.

My focus is on improving Bluetooth connection reliability and such improvement while worth are not my priority right now.

**Reply by DJm00n** [reply]
@darthcloud I'm happy, but seems my game (Syphon Filter SCES-01913) is not working with GameID system for some reason. :(
